### PR TITLE
Added new testcases tab to results window.

### PR DESCRIPTION
**Pull request by kt218**
Added new testcase tab that allows users to view the memory objects KLEE produced within the ktest files.
Added new processor "test_cases" to the pipeline.
Added new controller for testcases and modified CSS/HTML to results body.

### CODE REVIEW

**Comment by ccadar** [comment]
@kt218 you should rebase this now.

**Comment by ccadar** [comment]
This LGTM.  @andronat any comments?

File: src/klee_web/frontend/appjs/controllers.js [hunk 1]
```diff
@@ -416,3 +419,25 @@ controllers.controller('ResultTabsCtrl', [
...
+/*
+controllers.controller('TestcasesAccordionCtrl', [
+   '$scope',
+    function($scope){
+        $scope.status = {
+            isOpen: false
+        }
+    }
+]);
+ */
```
Comment (andronat) [inline src/klee_web/frontend/appjs/controllers.js]: Do we need this? If not, remove it.

File: src/klee_web/worker/processor/testcases.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import os
+import re
+
+from worker.processor.base import BaseProcessor
+
+
+class TestCaseProcessor(BaseProcessor):
```
Comment (andronat) [inline src/klee_web/worker/processor/testcases.py]: Can we have some tests for all these? e.g. an e2e test to check this functionality and the frontend would be awesome!

File: src/klee_web/frontend/appjs/controllers.js (hunk 1 above)
Comment (kt218) [reply src/klee_web/frontend/appjs/controllers.js]: I've removed this.

File: src/klee_web/worker/processor/testcases.py (hunk 2 above)
Comment (kt218) [reply src/klee_web/worker/processor/testcases.py]: Should these tests be inside test_runner.py and e2e.test.js?

Comment (kt218) [reply src/klee_web/worker/processor/testcases.py]: Tests have been added as e2e and runner tests.

These test klee-testcase runner for:
- runner producing the correct object and format
- matches the expected number of testcases
- matches expected number of mem objs per testcase
- order of testcases preserved by order of ktest file number

The e2e tests for:
- expected number of testcases appear through the pagination nav buttons
- 1 mem obj is shown for the first testcase

**Comment by ccadar** [comment]
Thanks, @kt218.  Can you also include a screenshot to quickly see how this looks like?

**Comment by kt218** [comment]
@ccadar Here's a gif of the testcases functionality.
klee_testcases

**Comment by ccadar** [comment]
Thanks, @kt218.  It's hard to follow in detail how the test cases are displayed via the GIF, but overall it looks like a nice addition. 

@andronat can you also take a look, merge and re-deploy the website when you can?  Thanks!

**Review by andronat — APPROVED** [review]
Let get this in! GJ @kt218!!
